src/Core/Search/SearchParameters.php

```diff
@@ -49,29 +49,64 @@ public function __construct(AdminFilterRepository $adminFilterRepository)
      */
     public function getFiltersFromRequest(Request $request, $filterClass)
     {
-        $filters = [];
+        $filters = new $filterClass();
```

Comment:
I'm more in favor of creating a new class with this behavior if possible... and deprecate this one.

Reply:
which class are you talking about? `SearchParamaters` or `Filters` ?
what behavior are you talking about exactly? because there are several changes in the file